content/doc/developer/forms/form-validation.adoc

```diff
@@ -45,6 +45,14 @@ public FormValidation doCheckThreads(@QueryParameter("value") final String meani
 
 This naming convention is how Jenkins wires up your form validation method. The return type also must be FormValidation, which offers various static factory methods to create an instance. Form validation can be of 3 different kinds: `ok`, `warning` or `error`. Validation message can either be represented in plaintext or use HTML markup. Several validations of different kinds can be reported wrapping then into one using aggregate factory method.
 
+=== To set up form validation within a Jelly template
+
+To configure form validation within a Jelly template to call doCheckXxx methods defined in a particular class in Jenkins, you'll indeed utilize the checkUrl and checkDependsOn attributes. Here's a breakdown of how these attributes are used:
+
+The checkUrl Attribute specifies the URL that Jenkins will request to perform form validation. Typically, this URL points to the Jenkins server, where it handles the validation request. The value entered in the input field will be checked via AJAX against this URL, and any errors encountered will be rendered under the text field. This URL is associated with the method responsible for performing the validation (doCheckXxx method) within your Descriptor class.
+
+The checkDependsOn Attribute defines the dependency between the field being validated and the field(s) whose values are required for validation. It specifies the ID(s) of the field(s) that the validation depends on. When the dependent field(s) change, Jenkins triggers the validation of the current field. Additionally, the fields specified in checkDependsOn are sent along with the form validation request.
```

Comment:
```suggestion
The `checkDependsOn` attribute defines the dependency between the field being validated and the field(s) whose values are required for validation.
It specifies the ID(s) of the field(s) that the validation depends on.
When the dependent field(s) change, Jenkins triggers the validation of the current field.
Additionally, the fields specified in `checkDependsOn` are sent along with the form validation request.
```
Suggestion to use one full sentence per line.
Suggestion to use code span for user entered/chosen attributes and lower case for "attributes" (to match usage elsewhere).

Reply:
yes understood from next time I will keep this in my notice, thank you for your help